<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tke_form_recalc_obsyag_gazu</class>
 <widget class="QWidget" name="tke_form_recalc_obsyag_gazu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>РОЗРАХУНОК перераховування обсягу газу</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_rozrahMonth">
     <property name="title">
      <string>Вибір місяця розрахунку</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_createMonthRozrah">
          <property name="text">
           <string>Створити</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/greplus.png</normaloff>:/pictures/pictures/greplus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteMonthRozrah">
          <property name="text">
           <string>Видалити</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/redminus.png</normaloff>:/pictures/pictures/redminus.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_calcValuesRozrahunok">
          <property name="text">
           <string>Розрахувати значення</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/calculating.png</normaloff>:/pictures/pictures/calculating.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_printAkt">
          <property name="text">
           <string>Друкувати АКТ</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/printer.png</normaloff>:/pictures/pictures/printer.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_printRozrahunok">
          <property name="text">
           <string>Друкувати РОЗРАХУНОК</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/printer.png</normaloff>:/pictures/pictures/printer.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_rozrahInMonth"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_budPokazn">
     <property name="title">
      <string>Побудинкові показники</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_copyValuesFromPrevMonth">
          <property name="text">
           <string>Копіювання показників з попередніх місяців</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/documents.png</normaloff>:/pictures/pictures/documents.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteButPokaznInRozrahunok">
          <property name="text">
           <string>Видалення показників у поточному розрахунку</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/button_cancel.png</normaloff>:/pictures/pictures/button_cancel.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_calcValuesBud">
          <property name="text">
           <string>Розрахунок значень</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/calculating.png</normaloff>:/pictures/pictures/calculating.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tableWidget_budPokazn">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_sum">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
